addons/mail/static/tests/owl/form_renderer_chatter_tests.js

```diff
@@ -75,13 +75,22 @@ QUnit.test('chatter updating', async function (assert) {
         id: 3, type: 'binary', mimetype: "application/x-msdos-program",
         name: "file3.txt", res_id: 5, res_model: 'partner'
     }];
-    this.data['res.partner'].records = [{
-        id: 1,
-        display_name: "first partner",
-    }, {
-        id: 2,
-        display_name: "second partner",
-    }];
+    this.data['res.partner'] = {
+        fields: {
+            message_attachment_count: { string: 'Attachment count', type: 'integer' },
+            message_ids: {string: "Messages", type: 'one2many', relation: 'mail.message'},
+            name: {string: "Name", type: 'char', default: ' '},
```

Comment:
Remove default if not needed, or put a real value if needed. This looks like a typo.

```suggestion
            name: { string: "Name", type: 'char' },
```